environment-pipelines/buildspec-plan.yml

```diff
@@ -14,6 +14,8 @@ phases:
   build:
     commands:
       - set -e
+      - echo "Cancelling any pending approvals for ${APPLICATION}-${ENVIRONMENT}-environment-pipeline"
+      - aws codepipeline get-pipeline-state --name "${APPLICATION}-${ENVIRONMENT}-environment-pipeline" | jq  --arg stage "Approve-${ENVIRONMENT}" -r '.stageStates[] | select(.stageName == $stage and .latestExecution.status == "InProgress") | .latestExecution.pipelineExecutionId' | xargs -I {} aws codepipeline stop-pipeline-execution --pipeline-name "${APPLICATION}-${ENVIRONMENT}-environment-pipeline" --pipeline-execution-id {}  --abandon --reason "Abandoning previous pipeline execution pending approval to run terraform plan"
```

Comment:
Can you break this down over a few lines to make it a bit more human-parsable? 
Perhaps:

- store output from get-pipeline-state in a variable
- Run it through jq to get the required vars
- Explicitly add those args to the end of the "stop-pipeline-execution" rather than piping them in through xargs

Reply:
Updated now.